App.js

```diff
@@ -15,14 +14,12 @@ import Tempo from "./components/Tempo";
 export default function App() {
 
   React.useEffect(() => {
-    // Ask location permission
-    requestLocationPermissions();
-    requestBluetoothPermissions();
-    requestNotificationPermissions();
+    requestAllPermissions();
   }, []);
 
   return (
     <NativeRouter>
+      <StatusBar style="dark" />
```

Comment:
Change: 
- show dark content of StatusBar
- new method to request permissions. more details further down in permissionsHandler file